uiplib/GUI.py

```diff
@@ -139,8 +144,11 @@ def set_wallpaper(self):
         image = self.images[self.index]
         change_background(image)
 
-    def refresh(self):
-        print("Refresh Clicked!")
+    def download(self):
+        pass
```

Comment:
I still haven't worked on it. Maybe call the scheduler to download new ones by flushing the very old ones or something like that.